src/Xml/Xslt/Configurator/functions.php

```diff
@@ -8,6 +8,8 @@
 use XSLTProcessor;
 
 /**
+ * TODO : Add support for callables : https://wiki.php.net/rfc/improve_callbacks_dom_and_xsl (either here or through a separate configurator)
```

Comment:
TODO : Add support for callable functions.

However, when I run the existing test-suite, it is not able to use the registered functions anymore:

> VeeWee\Xml\Exception\RuntimeException: Unable to apply the XSLT template
[ERROR] : xmlXPathCompOpEval: function function bound to undefined prefix php (1) on line 0,0
[ERROR] : runtime error: file /Users/verweto/Projects/github/veewee/xml/ line 7 element value-of (1) on line 0,0
[ERROR] : XPath evaluation returned no result. (1) on line 0,0



Template:

```xml
<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
    xmlns:php="http://php.net/xsl"
    xmlns:str="http://exslt.org/strings"
    xmlns:xsdl="http://www.w3.org/1999/XSL/Transform">
    <xsl:output method="text" omit-xml-declaration="yes" indent="no"/>
    <xsl:template match="/root">
        <xsl:value-of select="php:function('strtoupper', string(./hello))"/>
    </xsl:template>
</xsl:stylesheet>
```

Doc:

```xml
<root>
    <hello>World</hello>
</root>
```

Configuration:

```php
$processor->registerPhpFunctions();
```

Reply:
Indeed I can reproduce this on 8.4-dev, looks like a regression. I'll look into it today and get it fixed.

Reply:
Also, one (expected) difference on the master branch of PHP is that due to sharing code between XPath and XSL the error message got reworded:

```
1) VeeWee\Tests\Xml\Xslt\FunctionsTest::test_it_throws_exception_on_unkown_function
Failed asserting that exception message 'No callback handler "strtoupper" registered
' contains 'Not allowed to call handler 'strtoupper()''.
```

Reply:
I have identified and understood the problem, and will try to develop the solution this weekend. Thanks for finding this!

Reply:
I have pushed the fix for this in the development branch. Note that the branch is also rebased so it requires a full recompile.

Reply:
Tested it. Works like a charm! Thanks.